docs/userguides/console.md

```diff
@@ -52,3 +52,128 @@ Out[1]: 1
 ### Global Extras
 
 You can also add an `ape_console_extras.py` file to the global ape data directory (`$HOME/.ape/ape_console_extras.py`) and it will execute regardless of what project context you are in.  This may be useful for variables and utility functions you use across all of your projects.
+
+### Console Variables
```

Comment:
I think less is more for console namespace. We should just create a table with a small description; you can even just link to the manager Python docs for each item in the namespace.

Markdown table: https://markdown.land/markdown-table
Example doc to link to: https://docs.apeworx.io/ape/stable/methoddocs/managers.html#module-ape.managers.accounts

Reply:
The manager python doc does not have examples of what a user can do with them.

The purpose of this example is show the user how to use the console effectively. 

when @NotPeopling2day is going to do the tutorial. It would be nice to have something convenient to reference to.


We could move all the example to the managers. But it is not clear for a user to know to go to managers to look for sample uses of the console variables

Reply:
I am not saying move anything, just can link to it and keep it more consolidated here

Reply:
Look at the testing guide; it lists all the available fixtures but does not explain what accounts are in ape - just links to the part that does, so it is in one spot and easy to lookup and does not require keeping N number of docs about the same thing in-line with each other

Reply:
> Look at the testing guide; it lists all the available fixtures but does not explain what accounts are in ape - it is in one spot and easy to lookup but does keeps it focuses on the fixture-side of things rather than going into depth about what are accounts are in ape, etc

Chris and I talked a bit at length about this today, definitely recommend going the route of focusing on the right things in the right sections, and deep linking to other content where relevant